sql/upgrade-only/equipment_info.sql

```diff
@@ -1,27 +1,17 @@
 CREATE VIEW `equipment_info` AS
 SELECT
   `equipment`.`id` AS `id`,
-  JSON_OBJECT(
-    'id',`category`.`id`,
-    'name',`category`.`sub_category`,
-    'path',`category`.`category`
-  ) AS `category`,
   `equipment`.`manufacturer` AS `manufacturer`,
   `equipment`.`model` AS `model`,
   `equipment`.`description` AS `description`,
   `equipment`.`sku` AS `sku`,
   `equipment`.`quantity` AS `quantity`,
-  JSON_ARRAYAGG(
-    JSON_OBJECT(
-      'id',`tag`.`id`,
-      'name',`tag`.`tags`,
-      'category',`category`.`category`
-    )
-  ) AS `tags`,
+  JSON_OBJECT('id',`c`.`id`,'title',`c`.`title`,'parentId',`c`.`parent_id`) AS `category`,
+  JSON_ARRAYAGG(JSON_OBJECT('id',`tag`.`id`,'name',`tag`.`tags`,'category',`c`.`title`)) AS `tags`,
```

Comment:
`tag`.`tags` should be `tag`.`title`. Along with category, I initially committed this with an confusing column name